src/OrchardCore.Modules/OrchardCore.Media/Views/Admin/MediaApplication.cshtml

```diff
@@ -130,7 +130,7 @@
 @*Localizable Strings shared*@
 <input type="hidden" id="deleteFolder" data-title="@T["Delete media folder"]" data-message="@T["Are you sure you want to delete this folder?"]" />
 <input type="hidden" id="deleteMedia" data-title="@T["Delete media"]" data-message="@T["Are you sure you want to delete these media items?"]" />
-<input type="hidden" id="moveMediaMessage" value="@T["Are you sure you want to move the selected media to this folder?"]" />
+<input type="hidden" id="moveMedia" data-title="@T["Move media"]" data-message="@T["Are you sure you want to move the selected media to this folder?"]" data-ok="@T["Yes"]" data-cancel="@T["No"]" data-ok-class="btn-success" />
```

Comment:
Looking at the code, it seems the data-ok and data-cancel are not being used in the function call

Reply:
perhaps it's a resolve conflict issue :)

Reply:
AFAIK it works in many places

Reply:
No it did not work ;) It's because the text was the same as the default text. The data-ok was defined on the html element but never passed to the `confirmDialog` function. I modified everywhere in the code to use the new spread syntax so now all data attributes are passed through :)

Reply:
It worked for UnsafeUrl and RemoveUrl because properties were being passed explicitely. Anywhere else it would not work.